packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -202,12 +202,12 @@ const Bulletin = ({
             <span role="text">
               {offScreenText && (
                 <VisuallyHiddenText lang={lang}>
-                  {`${offScreenText},`}
+                  {`${offScreenText}, `}
```

Comment:
This is added so that in CSS disabled mode state is displayed with spaces instead of how it was before:
<img width="604" alt="Screenshot 2020-02-21 at 11 49 26" src="https://user-images.githubusercontent.com/17802955/75032870-76557000-54a1-11ea-9e55-41f8dca85a49.png">